src/components/HTMLEngineProvider/HTMLRenderers/MentionUserRenderer.js

```diff
@@ -33,11 +34,18 @@ function MentionUserRenderer(props) {
     // We need to remove the leading @ from data as it is not part of the login
     const loginWithoutLeadingAt = props.tnode.data.slice(1);
 
+    const accountID = _.first(PersonalDetailsUtils.getAccountIDsByLogins([loginWithoutLeadingAt]));
+
     const isOurMention = loginWithoutLeadingAt === props.currentUserPersonalDetails.login;
 
     return (
         <Text>
-            <Tooltip text={loginWithoutLeadingAt}>
+            <UserDetailsTooltip
+                accountID={accountID}
+                fallbackUserDetails={{
+                    displayName: loginWithoutLeadingAt,
```

Comment:
we don't need to pass anything else here since login is not shown if displayName is same as login 
https://github.com/Expensify/App/blob/bc7f68962a2251fd51d5eba1da24cafd90dd790d/src/components/UserDetailsTooltip/index.web.js#L38

and avatar is generated from optimistic `accountID` passed above in `UserDetailsTooltip` component here -
https://github.com/Expensify/App/blob/bc7f68962a2251fd51d5eba1da24cafd90dd790d/src/components/UserDetailsTooltip/index.web.js#L48